jupyterhub_singleuser_profiles/ui/src/SizesForm/components/SizesForm.css

```diff
@@ -44,3 +44,17 @@ div#sizes {
     display: grid;
     grid-template-columns: auto auto;
 }
+
+.DescDivider {
+    border-top: 1px solid black;
```

Comment:
I would replace the border with just a space, the border seems a bit too intrusive